boards/arm/96b_aerocore2/Kconfig.defconfig

```diff
@@ -0,0 +1,65 @@
+#  Copyright (c) 2020 Linaro Limited
+#  SPDX-License-Identifier: Apache-2.0
+
+if BOARD_96B_AEROCORE2
+
+config BOARD
+	default "96b_aerocore2"
+
+config UART_7
+	default y
+	depends on UART_CONSOLE
+
+config UART_1
+	default y
+	depends on SERIAL
+
+config UART_2
+	default y
+	depends	on SERIAL
+
+config UART_3
+	default y
+	depends on SERIAL
+
+config UART_7
+	default y
+	depends on SERIAL
```

Comment:
Wrap in an `if SERIAL..endif` instead of repeating `depends on SERIAL`. Do this for PWM and SPI too

Reply:
I was requested to make this change following https://github.com/zephyrproject-rtos/zephyr/pull/22666
https://github.com/zephyrproject-rtos/zephyr/pull/22095#discussion_r378376970

Reply:
#22666 was for single symbol if's. Here you have 4 UART instance symbols that all depend on the same thing.